gatorgrade/input/in_file_path.py

```diff
@@ -7,3 +7,28 @@ def parse_yaml_file(file_path):
     with open(file_path, encoding="utf8") as file:
         data = yaml.load_all(file, Loader=yaml.FullLoader)
         return list(data)
+
+
+def reformat_yaml_data(data):
+    """Reformat the raw data from a YAML file into a list of tuples."""
+    reformatted_data = []
+    data.pop(0)  # Removes the setup commands
+    loop_through_data(None, data[0], reformatted_data)
+    return reformatted_data
+
+
+def loop_through_data(path, data_list, reformatted_data):
```

Comment:
I think a more descriptive name of this function is needed to better describe what it does.